geomesa-convert/geomesa-convert-all/src/test/scala/org/locationtech/geomesa/convert/testing/EnrichmentCacheTest.scala

```diff
@@ -81,17 +86,17 @@ class EnrichmentCacheTest extends Specification {
           |      }
           |   }
           |   fields = [
-          |     { name = "id",     transform = "toString($0)"      }
+          |     { name = "id",     transform = "toString($1)"      }
```

Comment:
Is this shift between $0 and $1 something that is documented well?  

Is there anything we can do to help folks update their existing converters?

Reply:
this is just b/c i switched the converter used in this test from a 'test' implementation that was 0-based to the regular delimited-text converter, which is 1-based. this doesn't reflect any change in functionality.

Reply:
not sure about helping people upgrade existing converters...